src/components/item/ItemForm.vue

```diff
@@ -215,154 +242,181 @@ export default {
   <div
     v-if="edittingItemObj"
     class="item-form"
+    :class="`${globalTheme}-theme`"
   >
+    <!-- for future use
+    <header 
+      v-if="edittingItemObj.id"
+      class="header"
+    >
+      <PopUp
+        button-style-type="dots"
+        stop-propagation
+        position="lower-left"
+        class="additional-options-button"
+      >
+        <ButtonText
+          class="additional-option"
+          text="move to another list"
+          style-type="line"
+          small
+        />
+        <ButtonText
+          class="additional-option"
+          text="move to bin 🗑"
+          style-type="line"
+          small
+          @click="removeItem(edittingItemObj)"
+        />
+      </PopUp>
+    </header> -->
     <div class="text-fields">
       <InputCustom
-        ref="itemTitle"
-        label="item"
-        :model-value="edittingItemObj.title"
-        :placeholder="edittingItemObj.temporaryId ? $options.NEW_ITEM_PLACEHOLDER : ''"
+        class="title-input"
+        :model-value="itemName"
+        :placeholder="titlePlaceholder"
+        :is-focus="!edittingItemObj.id"
         @update:model-value="value => updateItemField('title', value)"    
       />
       <TextareaCustom
+        v-if="isDetailsTextareaShown"
         label="details"
         :model-value="edittingItemObj.details"
         @update:model-value="value => updateItemField('details', value)"
       />
+      <ButtonText
+        v-else
+        text="add details"
+        style-type="underline"
+        big
+        @click="toggleShowingStatus('detailsTextarea')"
+      />
     </div>
-    <div class="filters-section">
-      <div v-if="isAnyTagWithIdExist">
-        <div 
-          v-if="currentItemTags.length"
-          class="filters-container"
+    <div>
+      <div
+        v-if="areAddonsShown"
+        class="extra-content"
+      >
+        <TogglingBlock 
+          title="category"
+          caps
```

Comment:
I think it's better to rethink why something is setting by style-type and another by separate props - not consistent.
Could you create a ticket and put it next in queue?

Reply:
https://trello.com/c/fnfyZPGl/129-in-custom-form-elements-something-is-set-by-style-type-and-something-by-separate-props-need-to-rethink-as-it-is-not-consistent